### ISSUE TITLE
STM32F4 ENET MAC Support

### ISSUE THREAD

**Issue by roliver** [tock/tock-stm32#8]
Hi,

Is there any plan to add Ethernet MAC support when STM32F4 support is added?

Cheers,
Richard

**Comment by WizardUli** [comment]
Yes. I'd really like to have it in the future but the port is in its super-early beginnings right now.
